pkg/workloads/cortex/client/cortex/client.py

```diff
@@ -33,7 +38,133 @@ def __init__(self, env: str):
         """
         self.env = env
 
+    # CORTEX_VERSION_MINOR x5
     def deploy(
+        self,
+        config: dict,
+        cls=None,
+        pip_dependencies=[],
+        conda_dependencies=[],
+        force: bool = False,
+        wait: bool = False,
+    ):
+        """
+        Deploy an API.
+
+        Args:
+            config: A dictionary defining a single Cortex API. Schema can be found here:
+                → Realtime API: https://docs.cortex.dev/v/master/deployments/realtime-api/api-configuration
+                → Batch API: https://docs.cortex.dev/v/master/deployments/batch-api/api-configuration
+                → Traffic Splitter: https://docs.cortex.dev/v/master/deployments/realtime-api/traffic-splitter
+            cls: A Cortex Predictor class implementation. Not required when deploying a traffic splitter.
+                → Realtime API: https://docs.cortex.dev/v/master/deployments/realtime-api/predictors
+                → Batch API: https://docs.cortex.dev/v/master/deployments/batch-api/predictors
+            force: Override any in-progress api updates.
+            wait: Streams logs until the APIs are ready.
+
+        Returns:
+            Deployment status, API specification, and endpoint for each API.
+        """
+        project_dir = Path.home() / ".cortex" / "deployments" / str(uuid.uuid4())
```

Comment:
What does the `Path.home() / ".cortex" / "deployments"` do? Does it append the inlined strings on one another? Does the `/` act as an actual `+ "/" +` line of code?